agent/provision_agent.go

```diff
@@ -172,3 +175,18 @@ func (a *provisionAgent) deleteContainer(sc pfmodel.Container, lcs *pfmodel.Cont
 
 	return true, nil
 }
+
+func (a *provisionAgent) bootstrapProcess() bool {
+	log.WithFields(log.Fields{}).Warn("Starting bootstrap agent...")
+	filename := util.RandomString(10)
```

Comment:
I think we should remove this temporary filepath creation and moved it to within the function instead